apps/native-component-list/screens/ReactNativeCoreScreen.android.js

```diff
@@ -371,60 +373,28 @@ export default class ReactNativeCoreScreen extends React.Component {
 
   _renderWebView = () => {
     return (
-      <WebView
-        style={{ width: Layout.window.width, height: 250 }}
-        source={{
-          html: `
-          <h2>You can always use a WebView if you need to!</h2>
-          <p>
-            <h4>But don't the other components above seem like better building blocks for most of your UI?</h4>
-            <input type="text" placeholder="Disagree? why?"></input>
-            <input type="submit">
-          </p>
-          <p>
-            <a href="https://expo.io">expo.io</a>
-          </p>
-        `,
-        }}
-      />
-    );
-  };
-
-  _renderRow = renderRowFn => {
-    return <View>{renderRowFn && renderRowFn()}</View>;
-  };
-
-  _renderSectionHeader = (_, sectionTitle) => {
-    return (
-      <View style={styles.sectionHeader}>
-        <Text>{sectionTitle}</Text>
+      <View style={{ overflow: 'hidden' }}>
```

Comment:
could you add a brief comment here explaining why this is necessary? (maybe link to that GH issue you referenced)